chains/metadata.yaml

```diff
@@ -214,17 +242,43 @@ arcadiatestnet:
   deployer:
     name: Abacus Works
     url: https://www.hyperlane.xyz
-  displayName: Arcadia Testnet
+  displayName: Arcadia Testnet v2
   domainId: 1098411886
   isTestnet: true
-  name: arcadiatestnet
+  name: arcadiatestnet2
   nativeToken:
     decimals: 18
     name: Ether
     symbol: ETH
   protocol: ethereum
   rpcUrls:
     - http: https://rpc.khalani.network
+artelatestnet:
+  blockExplorers:
+    - apiUrl: https://betanet-scan.artela.network/api
+      family: blockscout
+      name: Artela Testnet Explorer
+      url: https://betanet-scan.artela.network
+  blocks:
+    confirmations: 1
+    estimateBlockTime: 2
+    reorgPeriod: 1
+  chainId: 11822
+  deployer:
+    name: Abacus Works
+    url: https://www.hyperlane.xyz
+  displayName: Artela Testnet
+  domainId: 11822
+  isTestnet: true
+  name: artelatestnet
+  nativeToken:
+    decimals: 18
+    name: Testnet Artela
+    symbol: ART
+  protocol: ethereum
+  rpcUrls:
+    - http: https://betanet-rpc1.artela.network
```

Comment:
interesting lol

Reply:
pre-emptively adding metadata for potential future testnets ;)